src/game_functions/game_structs.h

```diff
@@ -0,0 +1,38 @@
+#ifndef GAME_STRUCTS_H
+#define GAME_STRUCTS_H
+
+#include <stdbool.h>
+
+/* Holds data for single tile on the board. */
+typedef struct {
+    /* x position of tile. */
+    int x;
+    /* y position of tile. */
+    int y;
+
+    int bombs_around;
+    bool is_bomb;
+    bool is_flagged;
+    bool is_open;
+
+} Tile;
+
+/* Hold data about the minesweeper board. */
+typedef struct Board {
```

Comment:
As previously mentioned in past PRs, for line 21, instead of putting ``typedef struct Board``, it is sufficient to put ``typedef struct``.  You do this in line 7 so I assume that this was just an oversight.

Reply:
No, this was intentional. This is because line 35 ([line 35](https://github.com/SuchithSridhar/RTS-Minesweeper/pull/6/files/d888fb72bd4cf87a669b24186797b89aef3f156e#diff-3eb7c1107f85bb74c114efe23494724223e1218b4597b90398e10c48ed4bda16R35)), in the `Board` struct refers to itself. Since it's within the typedef, I cannot use the type (it's not been defined yet.)

Reply:
Gotcha